sdk/core/Azure.Core/CHANGELOG.md

```diff
@@ -1,11 +1,12 @@
 # Release History
 
-## 1.7.0-beta.1 (Unreleased)
+## 1.7.0 (2020-12-14)
 
 ### Added
```

Comment:
I believe this should be "New Features" according to the [guidelines](https://azure.github.io/azure-sdk/policies_releasenotes.html)